src/dfmpreview/package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "DfmPreview",
+    "displayName": "DfmPreview",
+    "description": "preview the DocFX Flavored Markdown in the vscode",
+    "version": "0.0.7",
+    "publisher": "928pjy",
```

Comment:
>928pjy [](start = 18, length = 6)

if we want to publish extension, I suggest use a more official account, not personal account